config/locales/views/admin/en.facility_statements.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  views:
+    facility_statements:
+      new:
+        instructions: Please review the following payment sources. A statement will be generated for each of the payment sources and an email with a link to the statement will be sent.
+        instructions_without_email: Please review the following payment sources. A statement will be generated for each of the payment sources.
+        no_orders: There are no completed orders that require statements.
+      index:
+        head: Statement History
```

Comment:
Not quite sure how this happened, but in the sidebar, the "Statement History" is appearing as "Send Notifications".